test/inductor/test_split_cat_fx_passes.py

```diff
@@ -292,6 +292,15 @@ def simple_split_cat_argspec3(x):
         def simple_split_cat_argspec4(x):
             return torch.cat(tensors=torch.split(x, 4, dim=1), dim=-2)
 
+        def simple_split_cat_node_dim_cat(x):
+            return torch.cat(torch.split(x, 4, dim=1), torch.tensor(1))
```

Comment:
@yanboliang (pinging you since you posted a review, but feel free to relay to whoever is most knowledgeable here)

This test case seems to have surfaced an unrelated torch.compile bug in CI:

```python
test/inductor/test_split_cat_fx_passes.py::TestSplitCatFxPasses::test_split_cat_merge - AssertionError: The values for attribute 'shape' do not match: torch.Size([2, 8, 4, 32, 16]) != torch.Size([2, 32, 32, 16]).
```

I cannot reproduce this in my machine if I pull out only snippet. Any recommendation on how to approach this?

Thank you